protos/dcs/tts/v0/tts.proto

```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+package dcs.tts.v0;
+import "dcs/common/v0/common.proto";
+option csharp_namespace = "RurouniJones.Dcs.Grpc.V0.Tts";
+option go_package = "github.com/DCS-gRPC/go-bindings/dcs/v0/tts";
+
+service TtsService {
+  rpc Transmit(TransmitRequest) returns (TransmitResponse) {}
+}
+
+message TransmitRequest {
+  // The text that is synthesized to speech and transmitted to SRS.
+  string text = 1;
```

Comment:
Should we add a second text field for the _plain_ text - i.e. no SSML tags, no transformations like FOUR TREE instead of 43? This plain text could then be emitted as an event for other clients to pickup (eg to present as text to the user for a11y use-cases)?

Reply:
That is a great idea. Have both SSML and text fields.

Reply:
Giving this a second thought, does this even make sense to have the raw text as an event? E.g. in the use-case of OverlordBot, I think it would make more sense if you execute an `OutText` in parallel right away, instead of relying on the TTS event that contains the raw text. Same goes for DATIS. 🤔

Reply:
I think there is a benefit to having an emitted event for some special use-cases such as clients for deaf and hard-of-hearing users as well as logging transmissions.

Reply:
Added in cf426c89de55f34943fc4a6070d7c279651e4229